src/pages/getapp.vue

```diff
@@ -59,6 +59,6 @@ export default {
 
 <style lang="scss">
 .getapp-page {
-  background: white;
+  background: #f7f7f7;
```

Comment:
I think you can remove the CSS here, as I rmb #f7f7f7 is the default color

Reply:
true, revert now